### ISSUE TITLE
memory warning

### ISSUE THREAD

**Issue by euhruska** [markovmodel/deeptime#17]
in the vampnet examples
```
hist = model.fit([X1_train, X2_train], Y_train ,
                         batch_size=batch_size,
                         epochs=nb_epoch,
                         validation_data=([X1_vali, X2_vali], Y_vali ),
                         verbose=0)
```
leads to 
```
tensorflow/python/ops/gradients_impl.py:108: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
```
It's just a warning, but might affect the performance

**Comment by amardt** [comment]
Hey,

I think the reason for the warning is the slicing of the eigenvalues and -vectors in the _inv function. I don't think that this will cause performance loss, but I changed the code to avoid this.  We have to test it first, though. See #18 

Best,

Andreas
